### PR TITLE
Implement `/team/{teamId}/pm-all` endpoint

### PR DESCRIPTION
**Pull request by Anupya**
## Description

Implement `message_all` = `/team/{teamId}/pm-all`

## After merge

Please check off `/team/{teamId}/pm-all` in issue description for #6.

### CODE REVIEW

File: berserk/clients/teams.py [hunk 1]
```diff
@@ -121,3 +121,13 @@ def search(self, text: str, page: int = 1) -> PaginatedTeams:
         path = "/api/team/search"
         params: Params = {"text": text, "page": page}
         return cast(PaginatedTeams, self._r.get(path, params=params))
+
+    def message_all(self, team_id: str, message: str):
+        """Send a private message to all members of a team. You must own the team.
```
Comment (benediktwerner) [inline berserk/clients/teams.py:126]: I don't believe this is true, one just has to be a leader with the right permissions.

File: berserk/clients/teams.py [hunk 2]
```diff
@@ -121,3 +121,13 @@ def search(self, text: str, page: int = 1) -> PaginatedTeams:
         path = "/api/team/search"
         params: Params = {"text": text, "page": page}
         return cast(PaginatedTeams, self._r.get(path, params=params))
+
+    def message_all(self, team_id: str, message: str):
```
Comment (benediktwerner) [inline berserk/clients/teams.py:125]:
```suggestion
    def message_all_members(self, team_id: str, message: str) -> None:
```

File: berserk/clients/teams.py (hunk 1 above)
Comment (Anupya) [reply berserk/clients/teams.py:126]: Is it wrong on the docs then? Screenshot 2023-10-22 at 9 29 06 PM

Comment (TBestLittleHelper) [reply berserk/clients/teams.py:126]: There was a recent change, so now Team Leaders have a more flexible permission system. Presumably you need the "Messages" team leder permission, which allows "Send private messages to all members". Might be similar outdated info in the api docs. (Granted "owner" was also incorrect, it was something any leader could do before)

image

Comment (Anupya) [reply berserk/clients/teams.py]: Updated! Please review 🙏

**Pull request merged**